src/Clock.tsx

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable no-console */
+import React, { useEffect, useState } from 'react';
+
+type ClockProps = {
+  clockName: string,
+};
+
+export const Clock: React.FC<ClockProps> = ({ clockName }) => {
+  const [today, setToday] = useState(new Date());
+  const [prevClockName, setNewClockName] = useState(clockName);
```

Comment:
it is simple with naming in the hooks. `[value, setValue]` - and you're good
```suggestion
  const [clockName, setClockName] = useState(clockName);
```